coreneuron/utils/randoms/nrnran123.cpp

```diff
@@ -76,33 +81,54 @@ using random123_allocator = coreneuron::unified_allocator<coreneuron::nrnran123_
  * shutdown. If the destructor calls cudaFree and the CUDA runtime has already
  * been shut down then tools like cuda-memcheck reports errors.
  */
-nrn_pragma_omp(declare target)
-philox4x32_key_t g_k{};
-nrn_pragma_omp(end declare target)
-nrn_pragma_acc(declare create(g_k))
-
+// nrn_pragma_omp(declare target)
+// philox4x32_key_t g_k_real{};
+// nrn_pragma_omp(end declare target)
+// nrn_pragma_acc(declare create(g_k))
```

Comment:
need to decide if we want to keep a version of the "old" code lying around for non-NVIDIA compilers

Reply:
> code lying around for non-NVIDIA GPU-offloading compilers. Clearly we need a non-NVIDIA CPU version that works.

I think so and I agree.